tasks/bes-reset.yml

```diff
@@ -1,2 +1,18 @@
 ---
-# TBD
\ No newline at end of file
+- name: Removing package
+  ansible.builtin.include_tasks: bes-remove.yml
+  
+- name: Getting default conf.
+  ansible.builtin.git:
+    repo: 'https://github.com/asa1997/ansible-role-bes-java.git'
+    dest: '{{ vagrant_home }}'
+    clone: yes
+
+- name: Resetting default conf.
+  ansible.builtin.copy:
+    src: "{{ vagrant_home }}/ansible-role-bes-java/defaults/main.yml"
+    dest: "{{ role_path }}/ansible-role-bes-java/defaults/main.yml"
+    remote_src: yes
+  
+- name: Installing package
+  ansible.builtin.include_tasks: bes-install.yml
```

Comment:
bes-install.yml not part of oah spec.Can  oah-install for the bes-oah role be leveraged?